etherman/etherman.go

```diff
@@ -273,7 +273,15 @@ func (etherMan *Client) sequenceBatches(opts *bind.TransactOpts, sequences []eth
 
 		batches = append(batches, batch)
 	}
-	return etherMan.PoE.SequenceBatches(opts, batches)
+
+	transaction, err := etherMan.PoE.SequenceBatches(opts, batches)
```

Comment:
there are other functions where this parse can be implemented such as the verifyBatch function

Reply:
I've added it in the last commit please check if this is the proper place. :)

Reply:
you also need to add it in the Approve